README.md

```diff
@@ -435,6 +434,16 @@ Taglib `AssetsHashResolverTag.java` zur Kompilierungszeit der JSP automatisiert
 <script defer src="<asset:url value='npm.jquery.js' />"></script>
 ```
 
+Während der Weiterentwicklung ist es sinnvoll das Caching zu deaktivieren. Wird das `demodata` Profil verwendet muss
+nichts weiter getan werden. Verwendest du das Profil nicht, kannst du das Caching mit folgenden application Properties
+deaktivieren:
+
+```properties
```

Comment:
Warum sind hier nicht die gleichen Properties gesetzt wir im `demodata`-Profil?

```
spring.resources.cache.cachecontrol.max-age=0
spring.resources.cache.cachecontrol.cache-public=false
spring.resources.cache.cachecontrol.cache-private=false
spring.resources.chain.strategy.content.enabled=false
spring.resources.chain.cache=false
```

Reply:
weil ich denke das `cache-public` und `cache-private` hier nicht nötig sind beschrieben zu werden.  
im `demodata` profil hingegen alles explizit stehen zu haben schadet nicht.

* `max-age`
  * header sagt aus wann wieder nachgefragt werden soll beim server nach neuem
* `cache-public`
  * z. B. CDNs dürfen cachen
* `cache-private`
  * z. B. der lokale Browser darf cachen (will man z. B. nicht für sensitive informationen. bei uns denke ich uninteressant, weil in den assets nichts sensitives ist. nur css und bisl client logik)
* `strategy.content.enabled`
  * content hash wird der ausgelieferten datei hinzugefügt oder nicht
* `chain.cache`
  * `false`: content hash wird bei jedem request neu berechnet
  * `true`: content hash wird einmalig berechnet

interessant bei der entwicklung ist also `max-age=0` und `strategy.content.enabled=false`. wenn man will kann `chain.cache` hier auch eingespart werden 🤔